src/components/data-exploration/differential-expression-tool/AdvancedFilteringModal.jsx

```diff
@@ -65,44 +67,55 @@ const AdvancedFilteringModal = (props) => {
     </Menu>
   );
 
+  const applyFilters = () => {
+    const formValues = form.getFieldsValue('filterForm').filterForm;
+    const formValuesFiltered = formValues.map(
+      ({ comparison, columnName, value }) => ({
+        comparison, columnName, value, type: 'numeric',
+      }),
+    );
+    onLaunch(formValuesFiltered);
+    onCancel();
```

Comment:
only call `onLaunch` here, not `onCancel`. 

If this is because we want to close the modal for both `onLaunch an `onCancel` that should be up to the callback handlers, not the one that executes the callbacks.